hasura.planx.uk/metadata/tables.yaml

```diff
@@ -202,8 +202,6 @@
                 _eq: x-hasura-lowcal-session-id
             - email:
                 _eq: x-hasura-lowcal-email
-            - submitted_at:
-                _is_null: true
```

Comment:
This feature requires that public users can select submitted sessions. This isn't actually an issue as queries for all sessions (e.g. resume) already have appropriate "WHERE" clauses.